install_ffmpeg.sh

```diff
@@ -75,47 +75,89 @@ if [ ! -e "$ROOT/x264" ]; then
   make install-lib-static
 fi
 
+if [ $(uname) == "Linux" && $BUILD_TAGS == *"debug-video"*]; then
```

Comment:
No support for VP8/VP9/H265 on Windows or Mac, then? That's a shame.

Reply:
Yes, no software support for these codecs on Windows, I think it's not worth the effort getting these dependencies working there. It's not feasible to use SW codecs on production, we are using them mostly for tests in lpms. For dev purposes, users are welcome to use Linux - or even latest WSL with CUDA support will work, I think?